xds/data/orca/v3/orca_load_report.proto

```diff
@@ -33,4 +34,8 @@ message OrcaLoadReport {
   // available, derived from the latest sample or measurement.
   map<string, double> utilization = 5
       [(validate.rules).map.values.double.gte = 0, (validate.rules).map.values.double.lte = 1];
+
+  // Total QPS being served by an endpoint. This should cover all services that an endpoint is
+  // responsible for.
+  double qps = 6 [(validate.rules).double.gte = 0];
```

Comment:
I'm fine with the move to `double`, that makes sense for stats, but I'm less enthusiastic about switching from RPS to QPS terminology as the way to distinguish the fields. They are synonyms, and a large chunk of the world uses RPS, whereas Google uses QPS and Envoy / xDS has significant influence from this. 

We should pick RPS / QPS as the xDS preferred standard, and put it in a style guide at the very least.

Reply:
As per our offline conversation, I've renamed `qps` to `rps_fractional`.